src/commands/utils/uploadUDF.test.ts

```diff
@@ -271,24 +283,164 @@ describe("uploadUDF", () => {
     });
 
     it("should handle upload errors properly", async () => {
-      const mockParams = {
-        environment: "env-123456",
-        cloud: "Azure",
-        region: "australiaeast",
-        artifactName: "test-artifact",
-        fileFormat: "jar",
-        selectedFile: tempJarUri,
-      };
+      // Simulate file not found error for the upload
```

Comment:
'file not found' would be coming from prepareUploadFileFromUri(), not a ResponseError from uploadFileToAzureStub(), wouldn't it? This comment doesn't match the actual exception path taken.

Perfectly valid and needed to have a test case over handling server-side rejecting, but the user-chosen file not existing locally case isn't that.

Should probably be separate test case for 'file open / preparing the request' error vs 'submitting artifact file to the upload route' error.

Reply:
changed comment